requirements/ci.in

```diff
@@ -1,3 +1,4 @@
+gitpython
```

Comment:
I see... we previously "fixed" this problem by adding `gitpython` in `docs.in`, but the script is called outside the tox env that uses that. Is it required there for another call (I suppose... building the docs themselves?)?

Reply:
> I suppose... building the docs themselves?

Yes

Reply:
Wonder whether once could move the call to the version script into the tox `docs` env..